packages/dds/merge-tree/src/ordinal.ts

```diff
@@ -20,11 +20,11 @@ export function computeHierarchicalOrdinal(
 	const ordinalWidth = 1 << (maxCount - actualCount);
 	let ordinal: string;
 	if (previousOrdinal === undefined) {
-		ordinal = parentOrdinal + String.fromCharCode(ordinalWidth - 1);
+		ordinal = parentOrdinal + String.fromCodePoint(ordinalWidth - 1);
 	} else {
-		const prevOrdCode = previousOrdinal.charCodeAt(previousOrdinal.length - 1);
+		const prevOrdCode = previousOrdinal.codePointAt(previousOrdinal.length - 1) ?? 0;
```

Comment:
Just a sanity check: is the `?? 0` behavior here comperable to what `charCodeAt` would have done on its own?